internal/tiltfile/tiltfile.go

```diff
@@ -104,6 +128,19 @@ func (t *Tiltfile) makeStaticBuild(thread *skylark.Thread, fn *skylark.Builtin,
 		return nil, fmt.Errorf("Argument 0 (dockerfile): %v", err)
 	}
 
+	var sba map[string]string
+	if buildArgs != nil {
+		d, ok := buildArgs.(*skylark.Dict)
+		if !ok {
+			return nil, fmt.Errorf("Argument 2 (build_args): expected dict, got %T", buildArgs)
```

Comment:
oh i see, these are 0-indexed.

...but also that's kinda dumb for human readability and i might 1-index them all? eh.